src/transform.h

```diff
@@ -68,6 +68,8 @@ int transform_validate(struct augeas *aug, struct tree *xfm);
  */
 int transform_load(struct augeas *aug, struct tree *xfm);
 
+int load_file(struct augeas *aug, struct lens *lens, const char *lens_name, char *filename);
```

Comment:
I think `load_file` is a little too low-level here.  It might be better to change the signature of `transform_load` to
```
int transform_load(struct augeas * struct tree *xfm, const char *file)
```
and change the implementation such that `transform_load` behaves the way it behaves today when you pass `NULL` for `file`, and if `file` is anything else, only actually load that file.